relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -539,7 +568,14 @@ mod tests {
 
     use relay_metrics::AggregatorConfig;
 
-    use crate::metrics_extraction::spans::extract_span_metrics;
+    use crate::metrics_extraction::spans::{extract_span_metrics, truncate_string};
+
+    #[test]
+    fn test_truncate_string_no_panic() {
+        let string = "ÆÆÆÆÆÆ".to_owned();
+        let truncated = truncate_string(string, 4);
```

Comment:
Would be nice to repeat this test with limit = 3 to see what happens if the limit is not on the string boundary, and with limits `string.len() - 1`, `string.len()`, `string.len() + 1` to cover the edge cases.